src/main/resources/scripts/sequence-files/purge_sequence_files.py

```diff
@@ -0,0 +1,48 @@
+#!/usr/bin/python
+import argparse
+import mysql.connector
+import os
+
+def list_sequence_files(host, user, password, database):
+    db = mysql.connector.connect(
+        host=host,
+        user=user,
+        password=password,
+        database=database
+    )
+    cursor = db.cursor()
+    # TODO: Should we double check this file doesn't exist in the actual table in case it was manually restored?
+    cursor.execute("SELECT DISTINCT file_path FROM sequence_file_AUD WHERE revtype=2")
```

Comment:
I see a big issue here. The revision number does not dictate what type of file it is.
With this line I assume you are trying to delete just the unzipped files, as they typically exist as type 2, and the .gz files exist as type 1.
This is not always the case, in this image you can see I did the click and drag to upload 2 files, file_1.fastq, and file_2.fastq.gz
![image](https://user-images.githubusercontent.com/16961365/220192356-bbf433d3-7d61-4d75-8a03-14d7e657bc7b.png)

Reply:
In the log here, you can see that file_1.fastq is under /55/1/ , file_2.fastq.gz is in /56/1/ and the unzipped file_2.fastq is in /56/2/

Reply:
I think there might be some confusion between REVTYPE and file_revision_number. My query is getting all the deleted files from the audit table.
You're right, I missed the compressed files. I think I had tunnel vision. I've updated the query to include all the file paths (compressed & uncompressed) that have been deleted. 
I also added an argument to the script to specify the sequence file base directory. 
Please see 47c5f00214b570cd20f2a15992b1472ebb104185.